kafka-ui-react-app/src/components/Topics/Topic/Messages/Filters/Filters.tsx

```diff
@@ -231,9 +231,13 @@ const Filters: React.FC<FiltersProps> = ({
         props.seekType = SeekType.TIMESTAMP;
       }
 
+      const isSeekTypeWithSeekTo =
+        props.seekType === SeekType.TIMESTAMP ||
+        props.seekType === SeekType.OFFSET;
```

Comment:
Correct me if I'm wrong: seekTo exists in the query only with this seek types?

Reply:
@Haarolean @iliax

Reply:
@NeiruBugz yep that's right